src/utils/api/getOrders.ts

```diff
@@ -0,0 +1,66 @@
+//
+// API call helper for getBidOrders & getAskOrders, not meant to be called directly (meant to be used inside the APIProvider)
+//
+
+import { API, IAPICall } from "./api";
+import { BASEURL } from "./constants";
+
+// http://localhost:7876/nxt?
+//   requestType=getBidOrders&
+//   asset=17554243582654188572&
+//   firstIndex=3&
+//   lastIndex=3
+
+// http://localhost:7876/nxt?
+//   requestType=getAskOrders&
+//   asset=17554243582654188572&
+//   firstIndex=3&
+//   lastIndex=3
+
+interface IGetOrdersParams extends IAPICall {
+  params: {
+    asset: number;
+  };
+}
+
+export async function getBidOrders(asset: number) {
```

Comment:
these two functions seem identical, so it might make sense to make a non-exported function getOrders which takes in a string parameter of 'Bid' or 'Ask' and uses it to create the `requestType` string needed. it's a small way of drying the code

Reply:
https://github.com/Up-North-Coding/jovian/issues/74